app/models/manageiq/providers/cloud_manager/orchestration_stack.rb

```diff
@@ -16,4 +19,13 @@ def self.create_stack(orchestration_manager, stack_name, template, options = {})
                  :cloud_tenant           => tenant,
                  :orchestration_template => template)
   end
+
+  def self.orchestration_stack_class_factory(orchestration_manager, template)
```

Comment:
AFAIk there is not a plan for making Tacker a separate provider, it's just another orchestration engine under OpenStack. Maybe if we would separate Orchestration as a provider? But I don't know about any plans of doing this.